GW2EIEvtcParser/EncounterLogic/Fractals/ShatteredObservatory/ShatteredObservatory.cs

```diff
@@ -49,5 +51,28 @@ protected static bool SetSuccessByBuffCount(CombatData combatData, FightData fig
             }
             return true;
         }
+
+        protected override void SetInstanceBuffs(ParsedEvtcLog log)
+        {
+            base.SetInstanceBuffs(log);
```

Comment:
Doesn't that require a full instance check? No being defeated on each encounter does not mean getting the achievement.
Perhaps only checking on Arkk makes more sense?

Reply:
I intended it as showing "still eligible on this boss", doesn't locating it inside ShatteredObservatory also work for full instance logs? I had no map logs to test it with.

Reply:
No, instance logs are barely supported, but fair enough.

Reply:
do i move it to arkk then?

Reply:
I think it'd be clearer to move it to Arkk for now, the UI won't be clear if the achievement is shown on Skorvald and Arts.